src/bucket/BucketIndex.h

```diff
@@ -37,6 +37,9 @@ class BucketManager;
 // BucketIndex abstract interface
 class BucketIndex : public NonMovableOrCopyable
 {
+  protected:
+    static const inline std::vector<PoolID> kEmptyVec = {};
```

Comment:
This doesn't follow the naming style. Also I don't think it needs to be a class member as it's only used in a single function (so could be defined right there).